### ISSUE TITLE
MultiSelect: Accessibility failure due to widget name

### ISSUE THREAD

**Issue by Joseph-Hitchcock** [IBM/carbon-components-react#1192]
When using the MultiSelect component, we're seeing the following error in DAP that's causing our accessibility tests to fail:

![image](https://user-images.githubusercontent.com/40905872/44108648-e43ae472-9faf-11e8-912c-7a33c599bb06.png)

After version `6.18.4` was released last Monday we were passing all of our accessibility tests with the same usage of MultiSelect, and since then this error has surfaced. We're currently on version `6.20.0`.

**Comment by joshblack** [comment]
Hi there @Joseph-Hitchcock! 👋 Thanks for taking the time to make this issue.

Really interesting to see the DAP error come up here, especially given the diff doesn't really change much: https://github.com/IBM/carbon-components-react/pull/1139/files

Could you share some more info from your project that might help us track down what's going on here? Thanks!

**Comment by Joseph-Hitchcock** [comment]
Hi @joshblack! Here's how we're creating the component in our code:

![image](https://user-images.githubusercontent.com/40905872/44116452-6a636028-9fc5-11e8-961e-76820cb90e0a.png)

As far as I can tell we're passing all of the required props, and it looks like the aria-label is being assigned internally from the label prop passed in. I haven't been able to do anything from this end to resolve the accessibility error.

**Comment by joshblack** [comment]
Thanks for the additional information! Really, really appreciate it 🙏 

Will take a look on our end, including running DAP here, to see what we can do 😄 

If you end up running into a fix in the meantime, we'll also 100% accept any contributions/PRs back into the project!

**Comment by Joseph-Hitchcock** [comment]
Hey @joshblack, I dug into the error a little bit more, and I found a fix that seems to resolve the accessibility issue. I read [here](https://www.w3.org/TR/wai-aria/#namecalculation) about what needs to be present for the name, and then looked into the usage of aria-label in the rendered HTML. Here's what we get right now from the MultiSelect snippet above:

![image](https://user-images.githubusercontent.com/40905872/44178597-efebf100-a0a7-11e8-9745-e39bdf82ac16.png)

I noticed that the element DAP pointed to was the parent `div` of the element with an aria-label, and that editing the HTML to give that element an aria-label resolves the error. I'm not sure what exactly caused this change looking at the diffs, but it looks like DAP wants the `listbox` element to have an aria-label.
